scripts/bench-lint.sh

```
@@ -0,0 +1,23 @@
+# The optional trailing args override the `vt` binary, e.g.
+#   scripts/bench-lint.sh ~/vault dev node /path/to/dist/cli.js
+# defaults to the installed `vt` on PATH.
+set -u
+repo="$1"; label="$2"; shift 2
+if [ "$#" -gt 0 ]; then vt_cmd=("$@"); else vt_cmd=(vt); fi
+cd "$repo" || exit 1
+files=$(find . -name '*.md' -not -path './.git/*' | wc -l | tr -d ' ')
+echo "== $label ($files md files on disk) =="
+for mode in "" "--no-suggestions" "--only drift"; do
+  s=$(python3 -c 'import time;print(time.time())')
+  # shellcheck disable=SC2086 -- $mode is intentionally word-split
```

Shellcheck (0.11.0)

[error] Couldn't parse this shellcheck directive. Fix to allow more checks.

(SC1073)